spec.md

```diff
@@ -487,6 +512,96 @@ The tags MUST be in lexical order.
 
 When using the `last` query parameter, the `n` parameter is OPTIONAL.
 
+##### Listing Referrers
+
+*Note: this feature was added in distibution-spec 1.1.
+Registries should see [Enabling the Referrers API](#enabling-the-referrers-api) before enabling this.*
+
+To fetch the list of referrers, perform a `GET` request to a path in the following format: `/v2/<name>/referrers/<reference>` <sup>[end-12a](#endpoints)</sup>.
```

Comment:
Reading through this and the examples below, I'm not actually clear on what string `<reference>` actually is. Media Type? digest? arbitrary value?